code/modules/food&drinks/recipes/tablecraft/recipes_soup.dm

```diff
@@ -0,0 +1,180 @@
+		/obj/item/weapon/reagent_containers/food/snacks/tofu = 2
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/milo
+
+/datum/table_recipe/bloodsoup
+	name = "Blood soup"
+	reqs = list(
+		/datum/reagent/blood = 10,
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tomato/blood = 2
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/blood
+
+/datum/table_recipe/slimesoup
+	name = "Slime soup"
+	reqs = list(
+			/datum/reagent/water = 10,
+			/datum/reagent/toxin/slimejelly = 5,
+			/obj/item/weapon/reagent_containers/glass/bowl = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/slime
+
+/datum/table_recipe/clownstears
+	name = "Clowns tears"
+	reqs = list(
+		/datum/reagent/water = 10,
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/banana = 1,
+		/obj/item/weapon/ore/bananium = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/clownstears
+
+/datum/table_recipe/mysterysoup
+	name = "Mystery soup"
+	reqs = list(
+		/datum/reagent/water = 10,
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/badrecipe = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/tofu = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/boiledegg = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge = 1,
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/mystery
+
+/datum/table_recipe/mushroomsoup
+	name = "Mushroom soup"
+	reqs = list(
+		/datum/reagent/consumable/milk = 5,
+		/datum/reagent/water = 5,
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/chanterelle = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/mushroom
+
+/datum/table_recipe/beetsoup
+	name = "Beet soup"
+	reqs = list(
+		/datum/reagent/water = 10,
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/whitebeet = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/cabbage = 1,
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/beet
+
+/datum/table_recipe/stew
+	name = "Stew"
+	reqs = list(
+		/datum/reagent/water = 10,
+		/obj/item/weapon/reagent_containers/glass/bowl = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tomato = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/cutlet = 3,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/potato = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/carrot = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/eggplant = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/mushroom = 1
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/soup/stew
+
+/datum/table_recipe/spacylibertyduff
```

Comment:
It's a bowl with a liquid and ingredients so I put it in that file, I can always make a ////////JELLIES//////// subcatergory.